tests/cypress/support/commands.js

```diff
@@ -40,3 +40,56 @@ Cypress.Commands.add('logInToWordPress', (username, password) => {
 		expect(authCookie).to.include('wordpress_logged_in_');
 	});
 });
+
+/**
+ * Mock and start a (fake) web monetization session.
+ */
+Cypress.Commands.add('startWebMonetization', () => {
+
+	cy.window().then((window) => {
+		startMonetization(window);
+
+		cy
+			.reload()
+			.then(window => {
+				startMonetization(window);
+			})
+	});
+});
+
+/**
+ * Mock and start a (fake) web monetization session.
+ *
+ * @param window
+ */
+function startMonetization(window) {
+	const doc = window.document;
+
+	// Shim the Web Monetization API: https://webmonetization.org/specification.html
+	if (!doc.monetization) {
+		doc.monetization = doc.createElement('div');
+	}
+	doc.monetization.state = 'started';
+
+	// Re-init Coil.
+	doc.dispatchEvent(new Event('coilstart'));
+
+	window.Cypress.monetized = true;
+
+	// Trigger the "user has paid $$$" event.
+	doc.monetization.dispatchEvent(new Event('monetizationstart'));
+}
+
+/**
+ * Stops fake monetization session. Must be ran after you are done testing
+ * with monetization as it will affect subsequent tests otherwise
+ */
+Cypress.Commands.add('stopWebMonetization', () => {
+	cy.window().then(window => {
+		const doc = window.document;
+		doc.monetization.state = 'stopped';
+		window.Cypress.monetized = false;
+		// Re-init Coil.
+		doc.dispatchEvent(new Event('coilstart'));
```

Comment:
Is this like a reset?

Reply:
Yes, I know this whole block looks sloppy however this is the only way I could get it to work.